test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -37,7 +37,7 @@ def hook(context):
     DEFAULT_CONFIG = {
         'localBackendUrl': 'https://localhost:9200/',
         'secureLocalBackendUrl': 'https://localhost:9200/',
-        'clientSyncTimeout': 10,
+        'clientSyncTimeout': 60,
```

Comment:
why increasing this?

Reply:
I changed it to check tests that have large size files. We can change it whenever needed.